mbuild/compound.py

```diff
@@ -1304,16 +1304,18 @@ def save(self, filename, show_ports=False, forcefield_name=None,
         # pad non periodic faces with .25nm buffers
         if box is None:
             box = self.boundingbox
+            box_vec_max = box.maxs.tolist()
+            box_vec_min = box.maxs.tolist()
```

Comment:
The issue was when we were just changing these values, the setter was not being called it seemed. box.lengths was not being updated. At least with the 2 single residue test case. The best way I could see would be to just unwrap that numpy array and perform the operations on it. Then pass it back as one numpy array which seemed to update box.lengths as well.